dpnp/backend.pyx

```diff
@@ -56,6 +56,7 @@ __all__ = [
 include "backend_logic.pyx"
 include "backend_manipulation.pyx"
 include "backend_mathematical.pyx"
+include "backend_searching.pyx"
 include "backend_sorting.pyx"
```

Comment:
```suggestion
include "backend_search.pyx"
include "backend_sort.pyx"
```

Reply:
These names are used in purpose to be compliant with original project/documentation. Are you really want to change them?
[Sorting, searching, and counting](https://numpy.org/doc/stable/reference/routines.sort.html)